ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/auth/audited/AuditedAuthenticator.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.auth.audited;
+
+import java.util.Optional;
+
+import org.apache.cassandra.auth.IAuthenticator;
+
+/**
+ * An {@link IAuthenticator} that provide custom fields to be used during authentication auditing.
+ */
+public interface AuditedAuthenticator extends IAuthenticator
```

Comment:
About names - I propose to align with naming conventions already in Cassandra and ecAudit.

So, how about IAuditAuthenticator?

Reply:
Yes, makes sense. Will change to IAuditAuthenticator